cookbook/configuration/web_server_configuration.rst

```diff
@@ -93,6 +93,18 @@ and increase web server performance:
         #     Options FollowSymlinks
         # </Directory>
 
+        # optionally disable the RewriteEngine for the asset directories
+        # which will allow apache to simply reply with a 404 when files are
+        # not found instead of passing the request into the full symfony stack
+          <Directory /var/www/project/web/bundles>
+            AllowOverride None
+            Order Allow,Deny
+            Allow from All
```

Comment:
These three lines should be removed (they are already covered by the `Directory` container for the `/var/www/project/web` path).